src/calypso-utils.js

```diff
@@ -9,24 +9,30 @@ const docblock = require('jest-docblock');
  */
 const shouldFormat = text => {
   const directives = docblock.parse(text);
-  return Object.keys(directives).indexOf('format') >= 0;
+  return Object.keys(directives).indexOf("format") >= 0;
```

Comment:
Can you test if `"format" in directives` would work here? You once said it doesn't work, but I'm using it successfully at many other places (eslines, reformat-files, ...)

Not terribly important, I only noticed it because the line got changed :)